File: app/src/main/java/com/example/java/CicilanResponse.java
package com.example.java;

import com.google.gson.annotations.SerializedName;
import java.util.ArrayList;
import java.util.List;

public class CicilanResponse {
    @SerializedName("status")
    private String status = "";

    @SerializedName(value = "total_sisa_cicilan", alternate = {"totalSisaCicilan", "sisa_cicilan"})
    private double total_sisa_cicilan = 0.0;

    @SerializedName(value = "total_harga_barang", alternate = {"totalHargaBarang", "harga_barang"})
    private double total_harga_barang = 0.0;

    @SerializedName(value = "total_dibayar", alternate = {"total_sudah_dibayar", "sudah_dibayar", "totalDibayar"})
    private double total_dibayar = 0.0;

    @SerializedName(value = "total_anggota_mencicil", alternate = {"anggota_mencicil", "total_anggota", "total"})
    private int total_anggota_mencicil = 0;

    @SerializedName("data")
    private List<CicilanAnggota> data = new ArrayList<>();

    public CicilanResponse() {
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public double getTotal_sisa_cicilan() {
        return total_sisa_cicilan;
    }

    public void setTotal_sisa_cicilan(double total_sisa_cicilan) {
        this.total_sisa_cicilan = total_sisa_cicilan;
    }

    public double getTotal_harga_barang() {
        return total_harga_barang;
    }

    public void setTotal_harga_barang(double total_harga_barang) {
        this.total_harga_barang = total_harga_barang;
    }

    public double getTotal_dibayar() {
        return total_dibayar;
    }

    public void setTotal_dibayar(double total_dibayar) {
        this.total_dibayar = total_dibayar;
    }

    public int getTotal_anggota_mencicil() {
        return total_anggota_mencicil;
    }

    public void setTotal_anggota_mencicil(int total_anggota_mencicil) {
        this.total_anggota_mencicil = total_anggota_mencicil;
    }

    public List<CicilanAnggota> getData() {
        return data != null ? data : new ArrayList<>();
    }

    public void setData(List<CicilanAnggota> data) {
        this.data = data;
    }
}

File: app/src/main/java/com/example/data/CicilanResponse.kt
package com.example.data

import com.google.gson.annotations.SerializedName

data class CicilanResponse(
    @SerializedName("status")
    val status: String? = null,

    @SerializedName("total_sisa_cicilan", alternate = ["totalSisaCicilan", "sisa_cicilan"])
    val total_sisa_cicilan: Double = 0.0,

    @SerializedName("total_harga_barang", alternate = ["totalHargaBarang", "harga_barang"])
    val total_harga_barang: Double = 0.0,

    @SerializedName("total_dibayar", alternate = ["total_sudah_dibayar", "sudah_dibayar", "totalDibayar"])
    val total_dibayar: Double = 0.0,

    @SerializedName("total_anggota_mencicil", alternate = ["anggota_mencicil", "total_anggota", "total"])
    val total_anggota_mencicil: Int = 0,

    @SerializedName("data")
    val data: List<CicilanAnggota>? = emptyList()
)

data class CicilanAnggota(
    @SerializedName("id")
    val id: Int = 0,

    @SerializedName("nama")
    val nama: String? = "",

    @SerializedName("nra")
    val nra: String? = "",

    @SerializedName("harga_barang", alternate = ["hargaBarang", "harga"])
    val harga_barang: Double = 0.0,

    @SerializedName("total_dibayar", alternate = ["totalDibayar", "dibayar"])
    val total_dibayar: Double = 0.0,

    @SerializedName("sudah_dibayar", alternate = ["sudahDibayar"])
    val sudah_dibayar: Double = 0.0,

    @SerializedName("sisa_cicilan", alternate = ["sisaCicilan", "sisa"])
    val sisa_cicilan: Double = 0.0,

    @SerializedName("cicilan_per_bulan", alternate = ["cicilanPerBulan"])
    val cicilan_per_bulan: Double = 0.0
)
